src/plugins/hetero/CMakeLists.txt

```diff
@@ -47,12 +47,6 @@ if(ENABLE_TESTS)
                 $<TARGET_PROPERTY:openvino::conditional_compilation,INTERFACE_INCLUDE_DIRECTORIES>)
 
         ov_set_threading_interface_for(${OBJ_NAME})
-
-        target_compile_definitions(${OBJ_NAME}
-            PRIVATE
-                USE_STATIC_IE IMPLEMENT_INFERENCE_ENGINE_PLUGIN IMPLEMENT_INFERENCE_EXTENSION_API
-                $<TARGET_PROPERTY:ngraph,INTERFACE_COMPILE_DEFINITIONS>
-                $<TARGET_PROPERTY:inference_engine_plugin_api,INTERFACE_COMPILE_DEFINITIONS>)
```

Comment:
looks like this change breaks Windows and we need to keep at least `USE_STATIC_IE`

Reply:
You were right, returned `USE_STATIC_IE IMPLEMENT_INFERENCE_ENGINE_PLUGIN IMPLEMENT_INFERENCE_EXTENSION_API`